### PR TITLE
Alter Toolbar Tooltips and some related plotting docs

### LINKED ISSUE #28604: Change Toolbar Home button tooltip WB plots
**Original reporter:** SANS group / Rob Dalgliesh

### Expected behavior
The tooltip for the Home Toolbar button on plots in workbench should be useful
### Actual behavior
"Center Display on Contents" isn't as useful as say "Reset Axes Limits"
### Steps to reproduce the behavior
Create a plot in WB and hover over the Toolbar buttons
### Platforms affected
WB 5.0

Consider if any of the other buttons tooltips need clarification. Can they be aligned with: https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/loading_and_displaying_data/06_formatting_plots.html#figureoptionsgear-png-ptions-menu

### PR DESCRIPTION
**Pull request by DanielMurphy22**
**Description of work.**

**Report to:** Rob Dalgliesh / SANS Group

**To test:**

**_Agree or disagree_** with my changes to the tooltips for toolbar buttons.
The linked issue requested the change of the Home button tooltip.

I have also tried to harmonise the Toolbar tooltips with the [Toolbar docs](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/loading_and_displaying_data/06_formatting_plots.html#plot-toolbar) and use Capital letter for key words.
If you have a suggestion for changing either, I'd be glad to discuss!

I have also changed a mistake on the docs toolbar image Right vs Left on Drag / Pan button, but no need to build the docs themselves! I've also added links to these [Keyboard Shortcuts](https://matplotlib.org/3.1.1/users/navigation_toolbar.html#navigation-keyboard-shortcuts) to the plotting docs. 

Build Mantid and run
```
ws = CreateSampleWorkspace()
plotSpectrum(ws,[0,1,2],waterfall = True)
plotSpectrum(ws,0)
```
and hover cursor over each toolbar button.

Fixes #28604

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: qt/applications/workbench/workbench/plotting/toolbar.py [hunk 1]
```diff
@@ -30,27 +30,27 @@ class WorkbenchNavigationToolbar(NavigationToolbar2QT):
     sig_waterfall_conversion = QtCore.Signal(bool)
 
     toolitems = (
-        ('Home', 'Center display on contents', 'mdi.home', 'on_home_clicked', None),
+        ('Home', 'Reset axes limits', 'mdi.home', 'on_home_clicked', None),
```
Comment (PhilColebrooke) [inline qt/applications/workbench/workbench/plotting/toolbar.py:33]: I think
```suggestion
        ('Home', 'Reset axis limits', 'mdi.home', 'on_home_clicked', None),
```
sounds better

File: qt/applications/workbench/workbench/plotting/toolbar.py [hunk 2]
```diff
@@ -30,27 +30,27 @@ class WorkbenchNavigationToolbar(NavigationToolbar2QT):
...
         ('Back', 'Back to previous view', 'mdi.arrow-left', 'back', None),
         ('Forward', 'Forward to next view', 'mdi.arrow-right', 'forward', None),
         (None, None, None, None, None),
-        ('Pan', 'Pan axes with left mouse, zoom with right', 'mdi.arrow-all', 'pan', False),
-        ('Zoom', 'Zoom to rectangle', 'mdi.magnify', 'zoom', False),
+        ('Pan', 'Pan: L-clk \nStretch: R-clk', 'mdi.arrow-all', 'pan', False),
+        ('Zoom', 'Zoom \n In: L-clk+drag \n Out: R-clk+drag', 'mdi.magnify', 'zoom', False),
```
Comment (PhilColebrooke) [inline qt/applications/workbench/workbench/plotting/toolbar.py:38]: I don't like L-clk and R-clk, I think just
```suggestion
        ('Pan', 'Pan: L-click \nStretch: R-click', 'mdi.arrow-all', 'pan', False),
        ('Zoom', 'Zoom \n In: L-click+drag \n Out: R-click+drag', 'mdi.magnify', 'zoom', False),
```
works fine.

File: qt/applications/workbench/workbench/plotting/toolbar.py [hunk 3]
```diff
@@ -30,27 +30,27 @@ class WorkbenchNavigationToolbar(NavigationToolbar2QT):
...
-        ('Print', 'Print the figure', 'mdi.printer', 'print_figure', None),
+        ('Grid', 'Grids on/off', 'mdi.grid', 'toggle_grid', False),
+        ('Save', 'Save image file', 'mdi.content-save', 'save_figure', None),
+        ('Print', 'Print image', 'mdi.printer', 'print_figure', None),
         (None, None, None, None, None),
-        ('Customize', 'Configure plot options', 'mdi.settings', 'launch_plot_options', None),
+        ('Customize', 'Options menu', 'mdi.settings', 'launch_plot_options', None),
         (None, None, None, None, None),
-        ('Create Script', 'Generate a script that will recreate the current figure',
+        ('Create Script', 'Generate Script to recreate the current figure',
```
Comment (PhilColebrooke) [inline qt/applications/workbench/workbench/plotting/toolbar.py:46]: I think having capital letters anywhere but at the beginning looks strange.
```suggestion
        ('Create Script', 'Generate script to recreate the current figure',
```

File: qt/applications/workbench/workbench/plotting/toolbar.py [hunk 4]
```diff
@@ -30,27 +30,27 @@ class WorkbenchNavigationToolbar(NavigationToolbar2QT):
...
          'mdi.script-text-outline', 'generate_plot_script', None),
         (None, None, None, None, None),
-        ('Fit', 'Toggle fit browser on/off', None, 'toggle_fit', False),
+        ('Fit', 'Open/close Fitting tab', None, 'toggle_fit', False),
         (None, None, None, None, None),
-        ('Offset', 'Change the curve offset percentage', 'mdi.arrow-expand-horizontal',
+        ('Offset', 'Adjust curve Offset %', 'mdi.arrow-expand-horizontal',
```
Comment (PhilColebrooke) [inline qt/applications/workbench/workbench/plotting/toolbar.py:51]:
```suggestion
        ('Offset', 'Adjust curve offset %', 'mdi.arrow-expand-horizontal',
```

File: qt/applications/workbench/workbench/plotting/toolbar.py [hunk 5]
```diff
@@ -30,27 +30,27 @@ class WorkbenchNavigationToolbar(NavigationToolbar2QT):
...
          'waterfall_offset_amount', None),
-        ('Reverse Order', 'Reverse curve order', 'mdi.swap-horizontal', 'waterfall_reverse_order', None),
+        ('Reverse Order', 'Reverse curve Order', 'mdi.swap-horizontal', 'waterfall_reverse_order', None),
```
Comment (PhilColebrooke) [inline qt/applications/workbench/workbench/plotting/toolbar.py:53]:
```suggestion
        ('Reverse Order', 'Reverse curve order', 'mdi.swap-horizontal', 'waterfall_reverse_order', None),
```

**Review by PhilColebrooke — CHANGES_REQUESTED** [review]

File: qt/applications/workbench/workbench/plotting/toolbar.py (hunk 1 above)
Comment (DanielMurphy22) [reply qt/applications/workbench/workbench/plotting/toolbar.py:33]: I think I prefer `axes` as it does affect both so I think the plural makes sense

File: qt/applications/workbench/workbench/plotting/toolbar.py (hunk 2 above)
Comment (DanielMurphy22) [reply qt/applications/workbench/workbench/plotting/toolbar.py:38]: happy to change to click!

File: qt/applications/workbench/workbench/plotting/toolbar.py (hunk 3 above)
Comment (DanielMurphy22) [reply qt/applications/workbench/workbench/plotting/toolbar.py:46]: Okay, I was only using to highlight what I saw as the main word, but this is not a big thing so I can change this!

File: qt/applications/workbench/workbench/plotting/toolbar.py (hunk 1 above)
Comment (PhilColebrooke) [reply qt/applications/workbench/workbench/plotting/toolbar.py:33]: That's fair, I'll allow it 😉

**Review by PhilColebrooke — APPROVED** [review]
Good stuff 😄

**Pull request merged**